AUTHORS.md

```diff
@@ -62,3 +62,4 @@ The following people have made contributions to this project:
 - [praerien (praerien)](https://github.com/praerien)
 - [Xin Zhang (zxdawn)](https://github.com/zxdawn)
 - [Yufei Zhu (yufeizhu600)](https://github.com/yufeizhu600)
+- [Taiga Tsukada (tsukada-cs)](https://github.com/tsukada-cs)
```

Comment:
Please insert your name in alphabetical order in the list.

Reply:
I wrote the test. However, I am not used to writing tests, so please check/advice me.

Reply:
This looks good. The test is run and passes.

Reply:
Thank you for your kind response.